package.json

```diff
@@ -10,10 +10,9 @@
     "enzyme-to-json": "^3.5.0",
     "node-sass": "^4.14.1",
     "react-bootstrap": "^1.0.0-beta.12",
```

Comment:
react-bootstrap should be removed since we are using `material-ui`

Reply:
now leave it as it's until the next milestone of the week, so we don't want to waste more time any more on deps, let's focus on landing page now @mwissem

Reply:
I really didn't notice it 😁, and since PR passed all the checks from the first time, so I don't wanna go back !